examples/README.md

```diff
@@ -53,3 +53,8 @@ API Examples
   read metadata and values for multiple tags at once.
 - [Subscription](tag/subscription): Demonstrates how to use the SystemLink Tag API
   to receive notifications when tag values change.
+
+### [Asset](asset)
```

Comment:
This link is missing a README.md. It would be a great place to explain that you need SystemLink Client installed.